opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/wsman-microsoft-windows-virtmem.properties

```
@@ -0,0 +1,32 @@
+##############################################################################
+##
+##  Please add report definition in a new line to make it easier
+##  for script based sanity checks
+##
+##################################################
+
+reports=microsoft.windows.virtmemfix
```

```suggestion
reports=microsoft.windows.virtmem1
```
Instead of fix, it may be better to name it as `virtmem1` through out this file.